testdata/sqllogictests/update.slt

```diff
@@ -0,0 +1,35 @@
+# Tests for updating rows from a table
```

Comment:
Could where add a test here with a `where` clause?

Reply:
sure. I will add more tests. It already has a where clause example

Reply:
oh missed that one. This seems good to go then.

Reply:
added another test case with multiple columns and multi update.